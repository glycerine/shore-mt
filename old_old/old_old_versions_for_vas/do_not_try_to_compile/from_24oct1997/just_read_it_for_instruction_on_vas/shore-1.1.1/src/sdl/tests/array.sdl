


module m1 {
	export attri;
	import opi_example;
	interface opi; // necessary
	interface attri {
	public:
		attribute long 		an_array[3];
        attribute ref<opi>  opi_array[3];
        attribute int  		int_array[3];
	};

}

module opi_example {
	export all;
	import m1;

	interface opi {
	private:
		// all combinations
	public:
		// some non-const operation
		void init();

	};

}

