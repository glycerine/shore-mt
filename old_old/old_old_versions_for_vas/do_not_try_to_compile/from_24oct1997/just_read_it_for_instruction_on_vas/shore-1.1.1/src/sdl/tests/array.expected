1. error in sdl_support.C:928 Invalid object
	called from:
	0) sdl_support.C:929

